homedir/.aliases

```diff
@@ -178,7 +183,7 @@ function common () {
 
 # aliases for common gh commands
 # github cli create draft pull request
-alias prc='gh pr create -df'
+alias prc='gh pr create -f'
```

Comment:
This should actually be maintained as `-df` to keep the initial PR as a draft. This is useful so that PRs aren't prematurely reviewed or merged.